nft_ingester/src/program_transformers/bubblegum/db.rs

```diff
@@ -97,10 +92,37 @@ where
         txn.execute(query)
             .await
             .map_err(|db_err| IngesterError::StorageWriteError(db_err.to_string()))?;
+    }
 
-        // Insert the audit item after the insert into cl_items have been completed
-        if let Some(audit_item) = audit_item {
-            cl_audits::Entity::insert(audit_item).exec(txn).await?;
+    // Insert the audit item after the insert into cl_items have been completed
+    if cl_audits {
+        let tx_id_bytes = bs58::decode(txn_id)
+            .into_vec()
+            .map_err(|_e| IngesterError::ChangeLogEventMalformed)?;
+        let audit_item_v2 = cl_audits_v2::ActiveModel {
+            tree: Set(tree_id.to_vec()),
+            leaf_idx: Set(change_log_event.index as i64),
+            seq: Set(change_log_event.seq as i64),
+            tx: Set(tx_id_bytes),
+            instruction: Set(Instruction::from_str(instruction)),
```

Comment:
Should this log a metric/error if the `Instruction::from_str` conversion results in `Instruction::Unknown`?

Reply:
Sure

Reply:
Not too critical from my perspective given that the instructions are known and we are well-aware of all changes. But defensiveness is good.